packages/cli/src/config/schema.ts

```diff
@@ -234,12 +234,11 @@ export const schema = {
 	},
 
 	executions: {
-		// By default workflows get always executed in the main process.
 		// TODO: remove this and all usage of `executions.process` when we're sure that nobody has this in their config file anymore.
 		process: {
 			doc: 'Own mode has been removed and is only here for backwards compatibility of config files. N8n will use main mode for executions unless `executions.mode` is set to `queue`.',
```

Comment:
```suggestion
			doc: 'Deprecated key, that will be removed in the future. Please remove it from your configuration and environment variables to prevent issues in the future.',
```